### PR TITLE
Support app package for semantic segmentation sample

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by Honry** [comment]
@fujunwei, @huningxin, PTAL, thanks!

File: node/examples/electron/semantic_segmentation/forge.config.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  },
+  makers: [
+    {
+      name: '@electron-forge/maker-zip'
+    }
+   ],
+  hooks: {
+    generateAssets: async () => {
+      // Copy WebNN libs
+      await fse.copy('../../../node_setup.js', 'node_setup.js', {overwrite: true});
```
Comment (fujunwei) [inline node/examples/electron/semantic_segmentation/forge.config.js:19]: Define a path variable of node like `let node_path = '../../../';`

**Review by fujunwei — APPROVED** [review]
LGTM with a open comment.

File: node/examples/electron/semantic_segmentation/forge.config.js (hunk 1 above)
Comment (Honry) [reply node/examples/electron/semantic_segmentation/forge.config.js]: Thanks, comment addressed.

**Comment by fujunwei** [comment]
Merge it, thanks.

**Pull request merged**